config/config.yaml

```diff
@@ -114,6 +114,9 @@ preprocessorConfig:
     applicationId: ${KAFKA_APPLICATION_ID:-kaldb_preprocessor}
     numStreamThreads: ${KAFKA_STREAM_THREADS:-2}
     processingGuarantee: ${KAFKA_STREAM_PROCESSING_GUARANTEE:-at_least_once}
+    additionalProps:
```

Comment:
We also need to add these props to json file also. Also, please update the KalDB unit tests that read this property.